src/dvc_render/image.py

```diff
@@ -20,17 +22,26 @@ class ImageRenderer(Renderer):
 
     EXTENSIONS = {".jpg", ".jpeg", ".gif", ".png"}
 
-    def partial_html(self) -> str:
+    def partial_html(self, output_dir=None, **kwargs) -> str:
```

Comment:
Do we need this arg? Afaik if `outs` was passed on DVC side, it should be present in `self.properties`:

https://github.com/iterative/dvc/blob/d0e05c4b66dc6e8b5e4ee46b622d42ab8b3fb01d/dvc/render/match.py#L48-L49

It makes more ense here if we expect this to be used outside DVC 🤔

Reply:
🤦

Reply:
After thinking a bit about it, i think that this was a proper approach. `out` contains information where to write the images. So in a sense it contains the same information as `datapoints.src` just without exact name of the file. When generating HTML via `render_html`, we can provide `index.html` file path, and it is the one path we are interested in the most. To be sure that our relpath is correct, we need to evaluate `index.html` path and calculate relative path to its `dirname`. Renamed `output_dir` to more informative `html_path`.